autofill/autofill-impl/src/main/java/com/duckduckgo/autofill/impl/ui/credential/selecting/AutofillSelectCredentialsDialogFragment.kt

```diff
@@ -128,7 +144,13 @@ class AutofillSelectCredentialsDialogFragment : BottomSheetDialogFragment(), Cre
             listItems = credentials,
         ) { selectedCredentials ->
 
-            pixelNameDialogEvent(Selected)?.let { pixel.fire(it) }
+            pixelNameDialogEvent(Selected)?.let {
+                appCoroutineScope.launch(dispatchers.io()) {
```

Comment:
using `appCoroutineScope` as the dialog is about to be dismissed and don't want to use that lifecycle